shared/src/server/health.rs

```diff
@@ -1,4 +1,11 @@
 use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub enum HealthCheckVersion {
+    V0(HealthCheckLog),
+    V1(HealthCheckLog),
```

Comment:
This will become (or similar)
```suggestion
    V1(DataPlaneDiagnostic),
```

Reply:
Was naming these strictly as a versioning on healthchecks not data-plane versions. V0 is the current healtcheck format V1 is what we're building? Messy to associate this with the existing enclaves V0/V1 versioning since it's not related.

Reply:
hmm fair. Was concerned the distinction wouldn't be obvious for future readers, but more likely a bias from our previous versioning pain.

Reply:
Yea I agree it's a bit confusing since we already have another v0-v1 thing going on in this project. Maybe a detailed comment would be sufficient